### ISSUE TITLE
🐛 [Umami] Doesn't support ARM64

### ISSUE THREAD

**Issue by DennisGaida** [runtipi/runtipi-appstore#3089]
### Store Application

Umami

### App version

1.40

### Description

Running on ARM64 is supposed to be supported, but logs show `exec /usr/local/bin/docker-entrypoint.sh: exec format error` which usually is a processor architecture problem.

Additionally version 1.40 seems pretty old - current is `2.11`. Maybe an image upgrade would already solve this problem.

### Steps to reproduce

1. Start App
2. Observe error

### App logs

```bash
exec /usr/local/bin/docker-entrypoint.sh: exec format error
exec /usr/local/bin/docker-entrypoint.sh: exec format error
exec /usr/local/bin/docker-entrypoint.sh: exec format error
```

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of RunTipi, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by meienberger** [comment]
Indeed, the image is still in v1. We are working on an automated update because it requires to run data migrations manually to upgrade to v2

**Comment by al-lac** [comment]
@meienberger Any update on this? 

As far as i have seen the migration needs to be done while still running the 1.40 version of umami.

No sure how once could structure the docker compose file to handle this scenario.

**Comment by steveiliop56** [comment]
We deprecated the old umami app and now there is the umami analytics app available in the appstore with the latest version. Unfortunately we cannot handle data migration ourselves so it will have to be done by users manually.
